portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,35 +14,98 @@
 
 package com.google.sps.servlets;
 
+import java.util.Map;
+import java.util.HashMap;
 import com.google.gson.Gson;
-import com.google.common.collect.ImmutableList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns inputted fun facts from users and stores previously inputted facts. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final ImmutableList<String> FRUIT_FACTS =
-    ImmutableList.of(
-      "A strawberry is not an actual berry", 
-      "Kiwis contain more Vitamin C than oranges", 
-      "The pineapple is actually a berry");
+  private Map<String, String> oldFacts = new HashMap<String, String>();
```

Comment:
super nit: usually constants are defined before non-static class variables like this Map

Reply:
Noted!